packages/react-reconciler/src/SchedulerWithReactIntegration.new.js

```diff
@@ -27,6 +27,7 @@ import {
 const {
   unstable_runWithPriority: Scheduler_runWithPriority,
   unstable_scheduleCallback: Scheduler_scheduleCallback,
+  unstable_scheduleMicrotaskCallback: Scheduler_scheduleMicrotaskCallback,
```

Comment:
I don't have a strong opinion on the API.

An alternative may be:

```js
scheduleCallback(
  MicrotaskPriority,
  performSyncWorkOnRoot.bind(null, root),
);
```

Reply:
I think it should just be queueMicrotask. It's just a literal polyfill for that function which is already a standard.